cmds/wifi/wifi.go

```diff
@@ -8,29 +8,16 @@ import (
 	"bytes"
 	"flag"
 	"fmt"
-	"io/ioutil"
 	"log"
 	"os"
 	"os/exec"
-	"regexp"
-	"strings"
 
-	"github.com/u-root/u-root/pkg/wpa/passphrase"
+	"github.com/u-root/u-root/pkg/wifi"
+	"github.com/u-root/u-root/pkg/wifi/wifiStub"
```

Comment:
just make it part of the package. If it's not used it won't get compiled inl

Reply:
Fixed